firmware/quintuna/CRIT/src/hw/hw_cans.cpp

```diff
@@ -4,7 +4,12 @@
 
 namespace hw::can
 {
-const CanBus can1{ &hcan1, [](const CanMsg *canMsg) { io::can1queue.pushRxMsgToQueue(canMsg); } };
+const CanBus can1{ &hcan1, [](const CanMsg *canMsg)
+                   {
+                       if (!io_canRx_filterMessageId(canMsg->std_id))
```

Comment:
consider this a hardware optimization, because w/best practices this is supposed to happen in the canrx task